lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscovery/SequentialSpec.hs

```diff
@@ -488,9 +488,13 @@ prop_changeIsOnlyKnownAfterGeneration (intPool, extPool) =
         ]
   where
     prop_addrsNotInInternalPool addrs =
-        map (\x -> (ShowFmt x, isNothing $ fst $ lookupAddress id x intPool)) addrs
+        map (\(x, s) ->
```

Comment:
So now `prop_addrsNotInInternalPool` doesn't really hold, because we have *used known internal addresses* in the return of `knownAddresses`. That's expected afais (that was the point of this PR, no?), so I adjusted that expectation.